pkg/api/validation/validation.go

```diff
@@ -4474,3 +4475,27 @@ func validateStorageNodeAffinityAnnotation(annotations map[string]string, fldPat
 	}
 	return policySpecified, allErrs
 }
+
+func validateSecurityContext(sc *api.SecurityContext) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if sc == nil {
+		return allErrs
+	}
+
+	if sc.AllowPrivilegeEscalation != nil && !*sc.AllowPrivilegeEscalation {
+		if sc.Privileged != nil && *sc.Privileged {
+			allErrs = append(allErrs, field.Forbidden(field.NewPath(""), "Cannot set `allowPrivilegeEscalation` to false and `privileged` to true"))
```

Comment:
This error doesn't look like `Forbidden`, it's more like `Invalid` configuration.